esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -62,6 +63,7 @@
 #define SDS_LOCK_TIMEOUT (5*60*1000) // 5 mins
 const unsigned CHECK_QUERY_STATUS_THREAD_POOL_SIZE = 25;
 const unsigned MAX_ZAP_BUFFER_SIZE = 10000000; //10M
+static const __uint64 getWUResultMaxSize = 10000000; //10M
```

Comment:
might be best if this was configurable, in case users are currently already relying on downloading files larger than this.